README.markdown

```diff
@@ -51,13 +51,22 @@ Add the class to agents you want to upgrade:
 include ::puppet_agent
 ~~~
 
+This installs the latest released version of Puppet from Puppet Collection 1.
+
+To upgrade with this module, first stand up a Puppet Server 2.1 master---which supports backward compatibility with Puppet 3 agents---and point the agent you want to upgrade at that master. Once you've confirmed the agent runs successfully against the new master, and thus the Puppet 4 language, apply the class to the agent and confirm that it checks back in after a successful upgrade. Further details on upgrading are available [here](http://docs.puppetlabs.com/puppet/4.2/reference/upgrade_major_pre.html).
```

Comment:
@jbondpdx @gguillotte I leaned towards fixing to a version rather than using latest; also not sure about what to call the link.